testing/rundeck/rundeck.initd

```diff
@@ -0,0 +1,23 @@
+#!/sbin/openrc-run
+
+# Rundeck init.d file for alpine linux.
+
+. /etc/rundeck/profile >/dev/null 2>&1
```

Comment:
this should be using conf.d settings, not sourcing a separate file in my opinion